### PR TITLE
Add input form to HelloWorld template [Fixes #205]

### LINKED ISSUE #205: HelloWorld template: add ability to update the message in the UI
## Summary

Currently there's no way to use the `update` function on the HelloWorld contract in the UI. We should provide a form input similar to the Coin template project.

## Motivation

The purpose of the UI is to provide users with an interactive dapp.

## Describe alternatives you've considered

A clear and concise description of the alternative solutions you've considered.

### PR DESCRIPTION
**Pull request by samajammin**
### Description of the Change
Add input form to HelloWorld template UI & functionality in app.js to update the contract storage.

### Benefits
Allows the user to interact with contract via the dapp UI interface.

### Verification Process
Tested locally & in production.

### Github Issues
#205

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
This would also need a read me update I guess?

**Comment by samajammin** [comment]
Good call - I've updated the README to reflect these template changes.

@ChrisChinchilla @javier-tarazaga please review when possible! Thanks.

File: packages/editor/templates/Hello World/README.md [hunk 1]
```diff
@@ -69,18 +69,27 @@ HelloWorld.prototype.init = function () {
 };
 ```
 
-Add other JavaScript boilerplate to create the instance of the `HelloWorld` object defined above, and show the HTML elements on the page:
+Add other JavaScript boilerplate to create the instance of the `HelloWorld` object defined above, show the HTML elements on the page, and bind the functions for interacting with the contract to the button defined in the HTML::
```
Comment (ChrisChinchilla) [inline packages/editor/templates/Hello World/README.md:72]:
```suggestion
Add other JavaScript boilerplate to create the instance of the `HelloWorld` object defined above, show the HTML elements on the page, and bind the functions for interacting with the contract to the button defined in the HTML:
```

**Comment by samajammin** [comment]
@javier-tarazaga re: your request:

> Please add a loader and disable the button until we get the receipt otherwise there is no feedback to the user that anything is happening meanwhile the tx is sent.

While I agree this is superior UX, I was just mimicking the behavior of the Coin template, which does not have a loader or disabling of the button. I suggest we merge this as-is & create an issue to add the behavior you describe to all templates.

**Pull request merged**